### ISSUE TITLE
Training from scratch, with pretrained VGG19 

### ISSUE THREAD

**Issue by zcemycl** [shepnerd/outpainting_srn#11]
I am trying to replicate the result of the dog dataset. To train from scratch, I rebuild it using PyTorch, and directly copy pytorch/model/{layer,loss}.py and pytorch/util/ from [inpainting_gmcnn](https://github.com/shepnerd/inpainting_gmcnn/tree/master/pytorch/model) to compute mrf loss. 

But whenever mrf_loss is included, the generator seems to quickly black out the image. I wonder if those files from inpainting_gmcnn is compatible to calculate mrf_loss? And if there is any detailed explanation about how the vgg19 features are used to compare?

Currently, the model is trained without mrf loss, the texture is bad. Below are my changed settings,
batch_size = 6
lrG = 1e-5
lrD = 5e-5
mrf_alpha = 0
others stay the same

My casual implementation: https://github.com/zcemycl/Pytorch_Outpainting_SRN
